www/http-parser/files/patch-Makefile.diff

```diff
@@ -0,0 +1,21 @@
+--- Makefile.orig	2018-04-30 11:44:19.000000000 -0300
```

Comment:
Nitpicking: I like to remove the arbitrary timestamp for the sake of next time (to prevent ending up with some bogus timestamp changes when edits are needed in patches).